docs/data/toolpad/studio/components/text-field/text-field.md

```diff
@@ -69,17 +69,25 @@ Disabled property shows the state of the component so that end user is aware tha
 
 ## Validation
 
+The validation props offer the option to create an interactive text field component for various scenarios. These are available as a checkbox configurations in the Toolpad Studio editor.
+
 ### isRequired
 
-isRequired is useful when the action can't be perfomed without a user provided text value.
+isRequired prop is useful when the action can't be perfomed without a user provided text value. It can be used to assign mandatory fields.
+
+{{"demo": "TextFieldIsRequired.js", "hideToolbar": true}}
 
 ### minLength
 
-A validation check on the minimum length of the input.
+A check on the minimum length of the input. In the below demo the input should be more than six characters long else it throws a validation error. It can be used to ensure passwords are long enough.
+
+{{"demo": "TextFieldMinlength.js", "hideToolbar": true}}
 
 ### maxLength
 
-A validation check on the maximum length of the input.
+A validation check on the maximum length of the input. The below demo shows a validation error when the input breaches the limit of ten characters. In combination with other props, it can be used for zip codes, phone numbers, etc.
```

Comment:
```suggestion
Shows an error message according to the maximum required length for the provided text value. In combination with other props, it can be used to enforce the length of zip codes or phone numbers, for example.

In the demo below, the input must be at least 10 characters long, otherwise a validation error is shown.
```